src/tokens/solana.tokenlist.json

```diff
@@ -68831,7 +68831,7 @@
       "symbol": "IRT",
       "name": "InJesterr Reward Token",
       "decimals": 0,
-      "logoURI": "https://raw.githubusercontent.com/AllClassInvestments/InJesterr-Reward-Token/main/IRT.png",
```

Comment:
How do I fix this error? Thanks

Reply:
"Your pull request only adds new tokens to the list. Any modification to existing
tokens will require manual review to prevent unwanted modifications." 

I am trying to update the image file used to represent IRT (InJesterr Reward Token) I will need a manual review, Thanks!